src/service/url-replacements-impl.js

```diff
@@ -49,6 +49,10 @@ const VARIANT_DELIMITER = '.';
 const ORIGINAL_HREF_PROPERTY = 'amp-original-href';
 const ORIGINAL_VALUE_PROPERTY = 'amp-original-value';
 
+/** A whitelist for replacements whose values should not be %-encoded. */
+/** @private @const {Object<string, boolean>} */
+const NOENCODE_WHITELIST = {'ANCESTOR_ORIGIN': true};
```

Comment:
Why are we not url encoding ancestor origins? It's never url safe...

Reply:
The current use case of this is to specify the domain to log to, not to put it in a query param, etc.